sdk/android/skin/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java

```diff
@@ -35,10 +35,8 @@
   public static WritableMap buildTimeChangedEvent(OoyalaPlayer player) {
     int playerDuration = player.getDuration();
     WritableArray cuePoints = Arguments.createArray();
-    Set<Integer> cuePointsPercentValues = player.getCuePointsInPercentage();
-    for (Integer cuePoint : cuePointsPercentValues) {
-      Double cuePointLocation = ((cuePoint / 100.0) * playerDuration) / 1000.0;
-      cuePoints.pushDouble(cuePointLocation);
+    for (Integer cuePoint : player.getCuePointsInMilliSeconds()) {
+      cuePoints.pushDouble(cuePoint * 0.001);
```

Comment:
I think we should either multiply everything on `0.001` or divide everything by `1000`

Reply:
maybe this will help with constants
https://docs.oracle.com/javase/6/docs/api/java/util/concurrent/TimeUnit.html

Reply:
`TimeUnit.HOURS.toMillis(1)`

Reply:
Either
```
   private final static int NUMBER_MILLISECONDS_IN_SECOND = 1000;
```
or
```
   private final static int NUMBER_MILLISECONDS_IN_SECOND = TimeUnit.SECONDS.toMillis(1);
```

Reply:
Feeling more convenient with the multiplying rather than division %)

Reply:
done